tools/bash/clone_herbert_branch.sh

```diff
@@ -25,20 +25,16 @@ if [ "${herbert_branch}" = "" ]; then
 fi
 
 if [[ -d "${HERBERT_DIR}" ]]; then
-    pushd "${HERBERT_DIR}" || exit 1
-    echo_and_run "git fetch -all --tags" &&
-    echo_and_run "git reset --hard \"${herbert_branch}\""
-    popd || exit 1
-
+    run_in_dir "git fetch --all --tags && git reset --hard \"${herbert_branch}\"" \
```

Comment:
the line break here is confusing with so much whitespace and escaped quotes in the command. Defining the command to execute as a variable would simply the call:
```
git_cmds = "git fetch --all --tags && git reset --hard \"${herbert_branch}\"
run_in_dir  "${git_cmds }" "${HERBERT_DIR}"
```